src/commands/diff.ts

```diff
@@ -0,0 +1,109 @@
+import { flags } from '@oclif/command';
+import * as diff from '@asyncapi/diff';
+import * as parser from '@asyncapi/parser';
+import { load } from '../models/SpecificationFile';
+import Command from '../base';
+import { ValidationError } from '../errors/validation-error';
+import { SpecificationFileNotFound } from '../errors/specification-file';
+
+export default class Diff extends Command {
+  static description = 'find diff between two asyncapi files';
+
+  static flags = {
+    help: flags.help({ char: 'h' }),
+    format: flags.string({
+      char: 'o',
+      description: 'output format',
+      default: 'json',
+    }),
+    type: flags.string({
+      char: 't',
+      description: 'the type of output',
+      default: 'all',
+    }),
+  };
+
+  static args = [
+    {
+      name: 'first-spec-file',
+      description: 'first spec path or context-name',
```

Comment:
I think help should be clear on explaining which file is checked against which. I mean for me it sounds like 1st is new and 2nd is old, but help should be clear about it with no doubts

Reply:
maybe `FIRST-SPEC-FILE` and `SECOND-SPEC-FILE` should be called differently? check maybe other diff tolls on how they make it clear

Reply:
Checked open api diff CLI, and they are using `openapidiff <old> <new>`. I will checkout other formats as well.

Reply:
`<old> <new>` sounds pretty straightforward

Reply:
Cool. I will go with `<old> <new>` then. Looked at a couple of more libraries, and they had the similar format, so that works.